src/s3b_the_roach_market.tw2

```diff
@@ -0,0 +1,280 @@
+<!-- If using debug menu shortcuts, prompts player to enter their name before beginning initial Roach Market sequence -->
+::RM-first-entrance-nameprompt
+{ (set: $playerName to (prompt: "Enter name:", ""))
+(if: (text: $playerName)'s length > 1)[Right, my name is $playerName. [[Moving on|RM-first-entrance]].<!-- what this code does is check the ending of the custom $playerName string to hopefully pluralize it properly -->{
+  (if: $playerName's last is "s")[
+    (set: $playerNames to ($playerName + "es"))
```

Comment:
To me, this implies a plural variable — multiple names. I think `$playerNamePlural` is more accurate, even if it's a bit wordier. Thoughts?

Reply:
ya, `$playerNames` isn't what I planned on keeping for the final variable name either. `$playerNamePlural` works!